src/ts/background/store/wallet.ts

```diff
@@ -1,16 +1,22 @@
 import { AnyAction, Reducer } from 'redux'
 
-export interface IAccount {
+export interface IWallet {
   locked: boolean,
   created: boolean,
   newPassword?: string,
   newPhrase?: string,
   newAccountName?: string
+  accounts?: IAccount[]
   currentAddress?: string
```

Comment:
as discussed, currentAccount replacing currentAddress and currentName is probably cleaner